central/networkgraph/flow/datastore/internal/store/postgres/store_test.go

```diff
@@ -54,27 +49,25 @@ func (s *NetworkflowStoreSuite) TestStore() {
 	defer pool.Close()
 
 	Destroy(ctx, pool)
-	store := New(ctx, pool, clusterId)
+	store := New(ctx, pool, clusterID)
 
-	deleteBefore := time.Now().Add(orphanWindow)
-	log.Info(deleteBefore)
-
-	networkFlow := &storage.NetworkFlow		{
+	networkFlow := &storage.NetworkFlow{
 		Props: &storage.NetworkFlowProperties{
 			SrcEntity:  &storage.NetworkEntityInfo{Type: storage.NetworkEntityInfo_DEPLOYMENT, Id: "a"},
 			DstEntity:  &storage.NetworkEntityInfo{Type: storage.NetworkEntityInfo_DEPLOYMENT, Id: "a"},
 			DstPort:    1,
 			L4Protocol: storage.L4Protocol_L4_PROTOCOL_TCP,
 		},
-		ClusterId: clusterId,
+		ClusterId: clusterID,
 	}
 
 	foundNetworkFlow, exists, err := store.Get(ctx, networkFlow.GetProps().GetSrcEntity().GetType(), networkFlow.GetProps().GetSrcEntity().GetId(), networkFlow.GetProps().GetDstEntity().GetType(), networkFlow.GetProps().GetDstEntity().GetId(), networkFlow.GetProps().GetDstPort(), networkFlow.GetProps().GetL4Protocol())
 	s.NoError(err)
 	s.False(exists)
 	s.Nil(foundNetworkFlow)
 
-	// Todo: The time and how it gets stored is hosing up the equal checks.  figure out best way to deal with that.
+	// Adding the same thing twice to ensure that we only retrieve 1 based on serial Flow_Id implementation
```

Comment:
Not yet since this store is custom and this store is currently the only one using a serial type as a PK meaning we can add the same item twice, but we only want to retrieve the most recent one.